src/main/java/gregtech/integration/groovy/GroovyScriptModule.java

```diff
@@ -273,6 +276,9 @@ public void onCompatLoaded(GroovyContainer<?> container) {
         container.objectMapperBuilder("oreprefix", OrePrefix.class)
                 .parser(IObjectParser.wrapStringGetter(OrePrefix::getPrefix))
                 .completerOfNamed(OrePrefix::values, v -> v.name)
+                .textureBinder(TextureBinder.of(i -> OreDictUnifier.getAll(new UnificationEntry(i)),
+                        TextureBinder.ofItem(),
+                        i -> String.format("![](${item('%s')}) %s", i.getItem().getRegistryName(), i.getDisplayName())))
```

Comment:
This is a bad idea since there are tons of items for each prefix

Reply:
Hmm, perhaps no texture inlays for oreprefix then?